01-syntax/04-high-order-functions/mother-child-age/ageDifference.js

```diff
@@ -1,12 +1,13 @@
 /* global ancestry */
```

Comment:
Use this one 
https://github.com/nesterone/sf-eng-init-gl-spring-2016/blob/master/01-syntax/04-high-order-functions/mother-child-age/ageDifference.js#L1

Hide all your work inside `ageDifference`